src/migrations/knexfile.ts

```diff
@@ -1,10 +1,16 @@
-import { DB_MIGRATIONS_TABLE_NAME, POSTGRES_URL } from 'back-end/config';
+import dotenv from 'dotenv';
+
+// Working directory changed to src/migrations so ensure env is loaded from correct path.
+dotenv.config({
+  debug: true,
+  path: '../../.env'
+});
 
 module.exports = {
   client: 'pg',
-  connection: POSTGRES_URL,
+  connection: process.env.POSTGRES_URL,
   migrations: {
-    tableName: DB_MIGRATIONS_TABLE_NAME,
+    tableName: 'migrations',
     directory: './tasks'
```

Comment:
TODO: I need to implement support for building migrations to JS, and running the JS directly to avoid this change

Reply:
So, I ended up figuring out the simpler solution is to properly resolve the REPOSITORY_ROOT_DIR variable in `back-end/config`. I am doing this by using the `find-up` package to find `package.json` and resolve its dirname. The build system stays the same for migrations.